openreview/venue/invitation.py

```diff
@@ -1509,6 +1509,236 @@ def set_public_comment_invitation(self):
+                                }
+                            },
+                            'ddate': {
+                                'param': {
+                                    'range': [ 0, 9999999999999 ],
+                                    'optional': True,
+                                    'deletable': True
+                                }
+                            },
+                            'signatures': ['${3/signatures}'],
+                            'readers': comment_stage.get_chat_readers(self.venue, '${5/content/noteNumber/value}'),
+                            'writers': [venue_id, '${3/signatures}'],
+                            'content': {
+                                'message': {
+                                    'value': {
+                                        'param': {
+                                            'type': 'string',
+                                            'maxLength': 1000,
+                                            'markdown': True
+                                        }
+                                    }
+                                }                                
+                            }
+                        }
+                    }
+                }
+            }
+        )
+
+        if comment_expdate:
+            invitation.edit['invitation']['expdate'] = comment_expdate
+
+        self.save_invitation(invitation, replacement=False)
+
+        self.client.post_invitation_edit(
+            invitations=self.venue.get_meta_invitation_id(),
+            signatures = [venue_id],
+            invitation = openreview.api.Invitation(
+                id = self.venue.submission_stage.get_submission_id(self.venue),
+                reply_forum_views = [
+                    {
+                        'id': 'discussion',
+                        'label': 'Discussion',
+                        'filter': f'-invitations:{venue_id}/{self.venue.submission_stage.name}${{note.number}}/-/Chat',
+                        'nesting': 3,
+                        'sort': 'date-desc',
+                        'layout': 'default',
+                        'live': True
+                    },
+                    {
+                        'id': 'committee-chat',
+                        'label': 'Committee members Chat',
+                        'filter': f'invitations:{venue_id}/{self.venue.submission_stage.name}${{note.number}}/-/Chat,{venue_id}/{self.venue.submission_stage.name}${{note.number}}/-/{self.venue.review_stage.name}',
+                        'nesting': 1,
+                        'sort': 'date-asc',
+                        'layout': 'chat',
+                        'live': True,
+                        'expandedInvitations': [f'{venue_id}/{self.venue.submission_stage.name}${{note.number}}/-/Chat']
+                    }
+                ]
+            )
+        )
+
+        emoji_chat_invitation = self.venue.get_invitation_id('Chat_Emoji')
```

Comment:
Instead of calling the invitation "Chat_Emoji" can we call it something more generic like "Chat_Reaction"